ros_buildfarm/common.py

```diff
@@ -271,6 +271,8 @@ def get_short_build_name(build_name):
 def get_short_os_name(os_name):
     os_name_mappings = {
         'debian': 'd',
+        'fedora': 'fc',
+        'rhel': 'el',
```

Comment:
Using two letters for those might be confusing. How about using just `f` and `r` instead?

Reply:
It looks fine on Jenkins. The short OS name is almost always combined with the short codename, so we're looking at, for example, 'uB' for Ubuntu Bionic. Here we'd see 'fc31' or 'el8', which are the short platform names used by an RPM:
* `kernel-5.4.8-200.fc31.x86_64.rpm`
* `kernel-4.18.0-80.11.2.el8_0.x86_64.rpm`